aggregator/demo/Dockerfile

```diff
@@ -18,21 +17,38 @@ RUN cd /tmp; ln -s /opt/jetty/webapps
 RUN chown -R jenkins /opt/jetty/logs
 RUN chown -R jenkins /opt/jetty/webapps
 
-ENV JENKINS_UC http://jenkins-updates.cloudbees.com
-COPY plugins.txt /tmp/files/
+WORKDIR /tmp/files
+
+# Prepare local Maven repo. Note that $JENKINS_HOME is a volume so we cannot populate it now.
+RUN mkdir repo-wc
+ADD repo/pom.xml repo-wc/pom.xml
+ADD repo/src repo-wc/src
+ADD repo/sometests repo-wc/sometests
+RUN chown -R jenkins.jenkins .
 USER jenkins
-RUN /usr/local/bin/plugins.sh /tmp/files/plugins.txt
+RUN echo '<settings><mirrors><mirror><id>central</id><url>http://repo.jenkins-ci.org/simple/repo1-cache/</url><mirrorOf>central</mirrorOf></mirror></mirrors><localRepository>/usr/share/jenkins/ref/.m2/repository</localRepository></settings>' > settings.xml
+RUN /usr/local/maven/bin/mvn -s settings.xml -f repo-wc install && /usr/local/maven/bin/mvn -s settings.xml -f repo-wc/sometests -Dmaven.test.failure.ignore clean install
 
-ADD repo /tmp/files/repo-wc
+ENV JENKINS_UC http://jenkins-updates.cloudbees.com
```

Comment:
> This URL?

Probably not required, but so far has seemed friendlier for forcing updates. Anyway yeah this is not new in this PR.